nb-configuration.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please add nb-configuration.xml to your global gitignore so these files do not end up in version control.